app/controllers/pages_controller.rb

```diff
@@ -1,22 +1,18 @@
 class PagesController < ApplicationController
-  append_view_path "app/views/pages"
+  append_view_path "pages"
```

Comment:
`app/views/pages` was already in the view path for this controller, but `pages` isn't.